### PR TITLE
fix asset startup error #81 #82

### PR DESCRIPTION
**Pull request by BishoyHanyRaafat**
@hal-8999-alpha lemme know if the issue is resolved

### CODE REVIEW

**Review by hal-8999-alpha — CHANGES_REQUESTED** [review]
@BishoyHanyRaafat thanks for doing this! Just ran it successfully, some small housekeeping things worth noting:

1. I went through and deleted all the assets one by one. Once I got to the last asset "Basic SQLAlchemy setup for Flask" after I deleted it I got the index out of range error. We should probably have a conditional specifically for the delete command that doesn't throw the error or suppresses it just to make it cleaner. Regardless, it handled it gracefully.

2. When there are no assets we should also probably display "No Current Assets". As of now it just displays a blank line. 

`User: list
1: Basic SQLAlchemy setup for Flask

User: delete
Name: Basic SQLAlchemy setup for Flask
Created: 5 days ago
Updated: 5 days ago
Type: CODE
Language: py

Are you sure you really want to delete this asset? This action cannot be undone. (y/n): y
Deleting asset...

An error occurred:
{IndexError('list index out of range')}


User: list
`

So basically the changes I'd request are:
1. Suppress/Remove index out of range error for the delete command.
2. When there are no assets display "No Current Assets".

Thanks for getting the ux functional!

**Comment by Bishoy-at-pieces** [comment]
@hal-8999-alpha can we merge this?

**Review by hal-8999-alpha — APPROVED** [review]
